app/konnect/gateway-manager/data-plane-nodes/upgrade.md

```diff
@@ -16,10 +16,64 @@ We recommend running one major version (2.x or 3.x) of a data plane node per con
 Read through the [{{site.base_gateway}} upgrade considerations](/gateway/latest/upgrade/) for the version that you're upgrading to.
 
 ## Upgrade a data plane node
-
 To upgrade a data plane node to a new version, follow these steps:
 
-1. Open {% konnect_icon runtimes %} **Gateway Manager**, choose a control plane,
+{% navtabs %}
+{% navtab Dedicated Cloud Gateways %}
+{% navtabs %}
+{% navtab Konnect UI %}
+1. Open [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), choose the Dedicated Cloud Gateways powered control plane,
+1. From the **Actions** drop-down menu, select **Update Cluster Config**
```

Comment:
```suggestion
1. From the **Actions** menu, select **Update Cluster Config**.
```
Feel free to ignore, this just aligns us with Google's style guide recommendations.